terraform/environment/dns.tf

```diff
@@ -0,0 +1,29 @@
+variable "subdomains" {
+  type = list(string)
+  default = [
+    "nginz-https",
+    "nginz-ssl",
+    "webapp",
+    "assets",
+    "account",
+    "teams"
+  ]
+}
+
+variable "create_spf_record" {
+  type = bool
+  default = true
+}
+
+module "aws-dns-records" {
+  count =  min(1, var.kubernetes_node_count)
```

Comment:
issue: this condition seems wrong. the question is, which variable should be defined, over in the env config, that leads to this module being instantiated. Maybe `length(root_domain) > 0`? Or a simple bool flag that indicates DNS creation?

Reply:
This was existing code in `hetzner-kubernetes`. Not sure what it's meant to do. Did you write it?